Extension/package.json

```diff
@@ -1115,8 +3759,8 @@
         "darwin"
       ],
       "binaries": [
-        "./bin/Microsoft.VSCode.CPP.Extension.darwin",
-        "./bin/Microsoft.VSCode.CPP.IntelliSense.Msvc.darwin"
+        "/home/griffin/vs/vscode-cpptools/Extension/bin/Microsoft.VSCode.CPP.Extension.darwin",
```

Comment:
This change to package.json needs to be reverted. Before making any changes to package.json, you need to make sure to undo the changes that occur when re-write the package.json with the runtime version of the file.

Reply:
Thanks for catching that! Is there any way we can automate this process for the future? I would like to investigate the possibility if it's worthwhile.

Reply:
I'm going to add a new option to our CMakeLists.txt for the language server that will point to your local clone of this instance and copy it into a disposable location so you don't have to run the extension on top of the local repository.  (this has been bugging me as well) :)

Reply:
Excellent! I will revert the changes as per Sean's comment and keep my eyes peeled for that change in the future.